client/call.go

```diff
@@ -48,6 +48,14 @@ func (c *Call) Method() proto.Method {
 	return c.Args.Method()
 }
 
+// ValueBytes returns a value as []bytes after a Get Call.
+func (c *Call) ValueBytes() []byte {
+	if gr := c.Reply.(*proto.GetResponse); gr.Value != nil {
```

Comment:
Feels slightly odd to put this here as it is specific to Get calls. I think we need a more holistic rethinking of the client API. This feels more like a bandaid.